content/sdk/10.js/00.ethers/00.ethers.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Getting Started"
+menuTitle: "JavaScript Versions"
+description: "Explore zkSync-ethers SDK versions"
+tags: "zksync, sdks, zkSync Era SDK, ethereum, era network, javascript versions"
+---
+
+These versions help developers interact with the zkSync Era blockchain. It covers most common
+operations developers require and provides a starting point for those new to the zkSync Era.
+
+::callout
+Take a look at [migration guide](/sdk/js/ethers/v6/migration) if you are migrating from `zksync-web3`.
+::
+
+### Prerequisites
+
+- Node: >=v18 ([installation guide](https://nodejs.org/en/download/package-manager))
+
+### Installation
+
+To install SDK for zkSync Era, run the command below in your terminal.
+
+::code-group
+
+```bash [ethers-v5]
```

Comment:
Is there a reason this is `ethers-v5` with the dash vs `ethers v6` that doesn't?

Reply:
edited